rdkit/Chem/__init__.py

```diff
@@ -101,24 +101,57 @@ def _getRDKitItem(self, i):
 
 
 rdchem.Mol.GetAtoms = lambda self: _GetAtomsIterator(self)
+rdchem.Mol.GetAtoms.__doc__ = """returns an iterator over the atoms in the molecule"""
 rdchem.Mol.GetBonds = lambda self: _GetBondsIterator(self)
+rdchem.Mol.GetBonds.__doc__ = """returns an iterator over the bonds in the molecule"""
 
 
 def QuickSmartsMatch(smi, sma, unique=True, display=False):
+  ''' A convenience function for quickly matching a SMARTS against a SMILES
+
+  Arguments:
+    - smi: the SMILES to match
+    - sma: the SMARTS to match
+    - unique: (optional) determines whether or not only unique matches are returned
+    - display: (optional) IGNORED
+
+  Returns:
+    a list of list of the indices of the atoms in the molecule that match the SMARTS  
+  
+  '''
   m = MolFromSmiles(smi)
   p = MolFromSmarts(sma)
   res = m.GetSubstructMatches(p, unique)
-  if display:
-    pass
   return res
 
 
 def CanonSmiles(smi, useChiral=1):
+  ''' A convenience function for canonicalizing SMILES
+
+  Arguments:
+    - smi: the SMILES to canonicalize
+    - useChiral: (optional) determines whether or not chiral information is included in the canonicalization and SMILES
+
+  Returns:
+    the canonical SMILES
+
```

Comment:
Is it worth noting that this internally calls `MolFromSmiles` and `MolToSmiles`? A user unaware of that called both `CanonSmiles` and `MolFromSmiles` on the same SMILES, which duplicates computational effort, so that note could prevent that.

Reply:
I'm going to leave this as is. I don't think it's unreasonable to expect that users understand that, aside from atom ordering, it doesn't matter if they construct a molecule from a SMILES or the canonical version of that SMILES

Reply:
Sure thing.

> I don't think it's unreasonable to expect that users understand that, aside from atom ordering, it doesn't matter if they construct a molecule from a SMILES or the canonical version of that SMILES

Concur with that logic.

My motivation was to help prevent users from unknowingly running `MolFromSmiles` twice, once explicitly and once opaquely via `CanonSmiles`, which duplicates computation.